packages/html/lib/jsx-runtime.js

```diff
@@ -49,7 +49,23 @@ const attrMap = {
     leafclassname: 'leafClassName',
 
     // Template
-    contenttemplate: 'contentTemplate'
+    contenttemplate: 'contentTemplate',
+
+    // Calendar
+    calendarheadertext: 'calendarHeaderText',
+    headercell: 'headerCell',
+    today: 'today',
+    weekend: 'weekend',
```

Comment:
these are not needed as they contain only one word and will be applied correctly as attributes